webapp/pages/password-reset.vue

```diff
@@ -20,3 +33,10 @@ export default {
   },
 }
 </script>
+<style lang="scss" scoped>
+img {
+  padding-left: 50px;
+  padding-right: 50px;
+  max-width: 200px;
+}
+</style>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-09T14:36:16Z" title="Wednesday, October 9th 2019, 4:36:16 pm +02:00">Oct 9, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -20,3 +25,10 @@ export default {
   },
 }
 </script>
+<style lang="scss" scoped>
+img {
+  padding-left: 50px;
+  padding-right: 50px;
+  max-width: 200px;
+}
+</style>
```


same here, if there is no img, why is there css?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-10T09:09:15Z" title="Thursday, October 10th 2019, 11:09:15 am +02:00">Oct 10, 2019</time>_

---


Yes, I just did not have time on Monday/Tuesday to clean up. Feel free to change that (although now I can also take care of this myself)